### PR TITLE
JW-584, JW-585: implemented pkt0 write

### PR DESCRIPTION
**Pull request by sodar**
- [x] ~Rebase from `jw-593` to `windows` after https://github.com/codilime/contrail-vrouter/pull/101 has been merged.~

### CODE REVIEW

File: windows/vr_pkt0.c [hunk 1]
```diff
@@ -58,28 +71,116 @@ Pkt0DispatchCleanup(PDEVICE_OBJECT DeviceObject, PIRP Irp)
...
 static NTSTATUS
 Pkt0DispatchWrite(PDEVICE_OBJECT DeviceObject, PIRP Irp)
 {
-    UNREFERENCED_PARAMETER(DeviceObject);
+    struct pkt0_context *ctx = VRouterGetPrivateData(DeviceObject);
+    PIO_WORKITEM work_item;
+    KIRQL old_irql;
 
-    /* TODO(sodar): Implement */
+    work_item = IoAllocateWorkItem(DeviceObject);
```
Comment (michal-clapinski) [inline windows/vr_pkt0.c]: Maybe use IoAllocateWorkItemEx.

File: windows/vr_pkt0.c [hunk 2]
```diff
@@ -58,28 +71,116 @@ Pkt0DispatchCleanup(PDEVICE_OBJECT DeviceObject, PIRP Irp)
...
+        Irp->IoStatus.Status = STATUS_INSUFFICIENT_RESOURCES;
+        IoCompleteRequest(Irp, IO_NO_INCREMENT);
+        return STATUS_INSUFFICIENT_RESOURCES;
+    }
 
-    Irp->IoStatus.Status = STATUS_SUCCESS;
-    Irp->IoStatus.Information = 0;
-    IoCompleteRequest(Irp, IO_NO_INCREMENT);
-    return STATUS_SUCCESS;
+    Irp->IoStatus.Status = STATUS_PENDING;
```
Comment (michal-clapinski) [inline windows/vr_pkt0.c]: I think this is useless

**Comment by sodar** [comment]
Retest this please

**Comment by michal-clapinski** [comment]
Retest this please

**Pull request merged**